project/design/schema.sql

```diff
@@ -0,0 +1,59 @@
+// TO DISTINGUISH BETWEEN ADMIN OR NORMAL USER
+CREATE TABLE IF NOT EXISTS usertype(
+	type varchar(20) PRIMARY KEY
```

Comment:
Good to have id as auto increment column. instead of user_tyoe, you can try with user_roles and have roles like admin, write, read, execute.

Reply:
done